app/controllers/concerns/sso_identity.rb

```diff
@@ -16,17 +22,17 @@ def current_user
   end
 
   def current_user_is_spo?
-    roles.include?('ROLE_ALLOC_MGR')
+    roles.include?(SPO_ROLE)
   end
 
+  # This role is granted to the service team to give them 'admin' priviledges
+  # such as editing teams and changing switch values
   def is_global_admin?
```

Comment:
Could this be a more consistent name for the admin method?

```suggestion
  def current_user_is_admin?
```

Given that we have the POM and SPO methods which already set out a convention of:
```
current_user_is_XXX?
```

Also note that I've suggested dropping 'global' from the name. I'm not sure if the addition of 'global' adds much more meaning to the 'admin' role name. And given the name of the constant `ADMIN_ROLE`, I'd rather we kept it consistent and simply went with 'admin'.

Reply:
Problem is that we have an 'ensure_admin_user' method that checks if you are an SPO....